### DISCUSSION TITLE
Quickly select the text from line a to line b, including these two lines.

### DISCUSSION THREAD

**Discussion by fengzyf** [WinEdt-Team/WinEdt-Support#21] [1.3 Ideas and features requests]
Dear Alex,

I would like to offer you a suggestion regarding the new features. 

When using  WinEdt, selecting the text from line a to line b (including these two lines) can be a bit cumbersome when the difference between a and b is large.

At present, when the cursor is positioned on line a, and the shortcut key Ctrl+G is used to pop up a window, enter the line number of line b, and  WinEdt quickly jumps to line b and selects the text in line b.

Therefore, I suggest adding a new optional option in the Ctrl+G pop-up window to support quick selection of text from the current line to the line b (including these two lines). Such functional improvements will greatly improve editing efficiency and make operations more convenient.

Thank you!

**Comment by Alex-Simonic** [comment]
This sounds easy enough. I will revise the dialog to accommodate an extra option to select the scope. You can check it out when the beta version is available.

There are also items in the Search -> Bookmarks menu to do similar things (Mark, Select from Mark, etc...). They are associated with qualifier+F11 shortcuts...
